js/src/radio-term-selector.js

```diff
@@ -405,6 +412,18 @@ class RadioTermSelector extends Component {
 				role="group"
 				aria-label={ groupLabel }
 			>
+				<div className={ 'editor-post-taxonomies__' + klass + '-terms-choice ' }>
+					<input
+						id={ `editor-post-taxonomies-${ klass }-term-${ -1 }` }
+						className={ 'editor-post-taxonomies__' + klass + '-terms-input components-radio-control__input' }
+						type="radio" // @helgatheviking
+						checked={ terms.length === 0 }
```

Comment:
@helgatheviking this needs replacing with a check to see that there are no terms assigned in this taxonomy to the current post. Retrieving `terms` in this component will do the trick

Reply:
I think I already have that covered [here](https://github.com/helgatheviking/Radio-Buttons-for-Taxonomies/blob/master/js/src/radio-term-selector.js#L319). I think what I might try is to add your extra link into the component and see how I get along!